app/controllers/admin/school_groups/school_onboardings_controller.rb

```diff
@@ -78,15 +77,20 @@ def produce_csv_row_manual(school)
         ]
       end
 
-      def for_selected(notice)
-        if (ids = params.dig(:school_group, :school_onboarding_ids))
-          @school_group.school_onboardings.find(ids).each do |onboarding|
+      def for_selected(message)
+        if onboardings.any?
+          onboardings.each do |onboarding|
             yield onboarding
```

Comment:
I know this isn't in your changes but I think, if possible, it would be useful to have a comment explaining what this yield is doing here.

Reply:
Thanks - it is my code from before though 😀 Happy to document!